dali/python/nvidia/dali/_conditionals.py

```diff
@@ -55,6 +55,30 @@ def check_nesting_support():
 
 
 if check_nesting_support():
```

Comment:
Would this work?:
```
if check_nesting_support():
    import nvidia.dali._conditionals_impl as _conditionals_impl 
else
    import nvidia.dali._conditionals_impl_legacy as _conditionals_impl
from _conditionals_impl import *        # noqa: F401, F403
from _conditionals_impl import _autograph        # noqa: F401
from _conditionals_impl import _DataNode        # noqa: F401
from _conditionals_impl import _Branch        # noqa: F401
from _conditionals_impl import _StackEntry        # noqa: F401
from _conditionals_impl import _ConditionStack        # noqa: F401
from _conditionals_impl import _cond_manager        # noqa: F401
from _conditionals_impl import _cond_true        # noqa: F401
from _conditionals_impl import _cond_false        # noqa: F401
from _conditionals_impl import _cond_merge        # noqa: F401
from _conditionals_impl import _verify_branch_outputs        # noqa: F401
from _conditionals_impl import _OVERLOADS        # noqa: F401
```

Reply:
I didn't. I am not sure why but I would like to move on since this is just a temporary thing anyway.